src/Extension/PhiremockProcess.php

```diff
@@ -47,6 +47,7 @@ class PhiremockProcess
      */
     public function start($ip, $port, $path, $logsPath, $debug)
     {
+        $path = realpath($path);
```

Comment:
Absolutly, we thought about the logFilePathe too but we needed a quick fix for the bin_path first. You are more familiar with your codes, so please decide it by yourself. =)